agent/consul/leader.go

```diff
@@ -822,32 +835,40 @@ func (s *Server) startACLReplication() {
 	}
 }
 
-type replicateFunc func(ctx context.Context, lastRemoteIndex uint64) (uint64, bool, error)
+type replicateFunc func(ctx context.Context, logger hclog.Logger, lastRemoteIndex uint64) (uint64, bool, error)
 
 // This function is only intended to be run as a managed go routine, it will block until
 // the context passed in indicates that it should exit.
 func (s *Server) runACLPolicyReplicator(ctx context.Context) error {
-	s.logger.Printf("[INFO] acl: started ACL Policy replication")
-
-	return s.runACLReplicator(ctx, structs.ACLReplicatePolicies, s.replicateACLPolicies)
+	policyLogger := s.aclReplicationLogger(structs.ACLReplicatePolicies.SingularNoun())
```

Comment:
I'm sorry I missed this when I left a bazillion comments about losing your `item_type`. Will remove.